packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript

```diff
@@ -1,24 +1,19 @@
-if [[ "$(grep -Po 'ID=\K[^"]+' /etc/os-release)" == "debian" ]]; then
-    fancy_message error "This script doesn't work on Debian"
-    cleanup # Trigger pacstall cleanups
-    exit 1
-fi
 name=an-anime-game-launcher-bin
 pkgname=an-anime-game-launcher
-version=2.2.0
+version=2.2.2
 description="An Launcher for a specific anime game with auto-patching, discord rpc and time tracking"
-url="https://kaifa.ch/an_anime_game_launcher_220.tar.xz"
-hash="fcc5c715fb1bdd1f272d91ad61ec9f72a83218e2221f4ec06aa42226ee03899b"
+repology=("project: an-anime-game-launcher")
+url="https://aagl.launcher.moe/dl/an_anime_game_launcher_${version//./}.tar.xz"
+hash="8b741b767cf297f645aacaf1055684bd86b32bb533cde3acfaad67eb3a29b35b"
 maintainer="Marie Piontek <marie@kaifa.ch>"
 gives="${pkgname}"
 breaks="${pkgname} ${pkgname}-deb ${pkgname}-app ${pkgname}-git"
 
-depends="libappindicator3-1 tar unzip xdelta3 cabextract git"
+depends="libayatana-appindicator3-1 tar unzip xdelta3 cabextract git"
 
 optdepends=(
     "mangohud: FPS Hud/GUI"
     "gamemode: Game Optimizations"
-    "switcheroo-control: Hybrid GPU Support"
     "vkbasalt: Required to use custom shaders (install this and reshade-shaders-git)"
     "reshade-shaders-git: Required by vkBasalt config files (install this and vkbasalt)"
```

Comment:
Since `reshade-shaders-git` doesn't exist on the `apt` repos it fails on develop branch
```suggestion
    #"reshade-shaders-git: Required by vkBasalt config files (install this and vkbasalt)"
```
`vkbasalt` does exist in the repos tho. Is `reshade-shaders-git` really needed?

Anyway, we could inplement a optpacdeps of sorts, but for now this doesn't work on 1.7.3

Reply:
reshade-shaders-git is indeed needed for vkbasalt to be used in the game as all shaders included in the launcher use reshade shaders

Reply:
Very well.

For now, since 1.7.2 has a bug that it doesn't install optdeps properly, it doesn't matter.
I'll see what I can do with optional pacdeps. We would need someone to make a reshade-shaders-git pacscript tho.

Reply:
Yea, having someone make the package shouldn't be hard as it's a very basic one
[PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=reshade-shaders-git)